### PR TITLE
Add README and ONNXOpCoverage doc back

### PR DESCRIPTION
**Pull request by houseroad**
Just manually updated the coverage doc.

### CODE REVIEW

File: caffe2/python/onnx/README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+# Usage
+
+* [ONNX to Caffe2](https://github.com/onnx/tutorials/blob/master/tutorials/OnnxCaffe2Import.ipynb)
+* [Caffe2 to ONNX](https://github.com/onnx/tutorials/blob/master/tutorials/Caffe2OnnxExport.ipynb)
+* [other end-to-end tutorials](https://github.com/onnx/tutorials)
+
+# Installation
+
+onnx-caffe2 is installed as a part of Caffe2.
+Please follow the [instructions](https://caffe2.ai/docs/getting-started.html) to install Caffe2.
```
Comment (yinghai) [inline caffe2/python/onnx/README.md:13]: Awesome!

**Review by yinghai — APPROVED** [review]
Thanks a lot. However do we update the coverage? :)

File: caffe2/python/onnx/README.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
+onnx-caffe2
```
Comment (bddppq) [inline caffe2/python/onnx/README.md]: remove this?

**Comment by houseroad** [comment]
I did update the coverage, just polished again, some "OK" is missing.

**Pull request merged**